buildscripts/grpc-java-artifacts-aarch64/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM dockcross/manylinux2014-aarch64
```

Comment:
I'd like to make the protoc build work with the original grpc-java-artifacts build, but so far I haven't yet figured out how.
(see https://github.com/grpc/grpc-java/pull/8096#discussion_r619220202)